services/forest/src/util/errorHandler.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Wrapper to catch errors from every endpoint. When an error occurs in a lower layer,
+ * depending on the error it is caught and organized to be thrown again an then log it here,
+ * or it's just handled here.
+ *
+ * @param {Object} logger logger instance.
+ * @param {Function} callback the function to execute.
+ */
+module.exports = logger => callback => (
+  async (req, res, next) => {
+    try {
```

Comment:
Please add the info logs added to `services/backend/src/util/errorHandler.js`

Reply:
Done